src/GT.Core/Services/Impl/GTListingService.cs

```diff
@@ -398,11 +400,11 @@ public async Task<bool> ExistsByIdAsync(string id)
 			var newListingEntity = new Listing();
 
 			newListingEntity.Id = Guid.NewGuid().ToString();
-			newListingEntity.ListingTitle = listingDTO.ListingTitle!;
-			newListingEntity.Description = listingDTO.Description;
+			newListingEntity.ListingTitle = listingDTO.ListingTitle! == null ? null : listingDTO.ListingTitle.Trim();
```

Comment:
Does exclamation mark (" ! ") do anything in updated code?

Reply:
We were unsure about it´s purpose and decided to leave it as is.